lib/jabbax/document/error_source.ex

```diff
@@ -1,7 +1,13 @@
 defmodule Jabbax.Document.ErrorSource do
   defstruct [:pointer, :parameter]
 
-  def from_attribute(attribute) do
-    %__MODULE__{pointer: "/data/attributes/#{attribute}"}
+  def from_attribute(attribute, prefix \\ []) do
```

Comment:
I'm wondering whether it wouldn't make more sense for this function to accept two types of arguments - a list and an atom/string. The "prefix" here is really a path to the attribute, not a namespace or modifier... We could then have:


```elixir
def from_attribute(path) when is_list(path) do
  pointer = "/data/attributes/#{Enum.join(path, "/")}"
  %__MODULE__{pointer: pointer}
end

def from_attribute(attribute), do: from_attribute([attribute])
```

And use it like:

```elixir
ErrorSource.from_attribute(:name) # => %ErrorSource{pointer: "/data/attributes/name"}
ErrorSource.from_attribute([:address, :city]) # => %ErrorSource{pointer: "/data/attributes/address/city"}
```

WDYT?

Reply:
Ok, I changed it. But now I think it's a bit weird to pass a list to `from_attribute` function. Perhaps the one accepting a list should be named `from_attributes`?

Reply:
The trick is to think of it as an attribute _path_, which is a list. If you call the argument `attribute_path` then it makes sense I think - while the old implementation could use the name `attribute` for the argument.